integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/AbstractE2ETest.java

```diff
@@ -42,11 +46,16 @@ protected void setupTest(TestContext testContext) {
         createDefaultEnvironment(testContext);
     }
 
-    @AfterMethod
+    @AfterMethod(dependsOnMethods = { "validateResourceTags" })
```

Comment:
What's the benefit of having 2 separate @AfterMethod linked together with depends on, rather then having a single Aftermethod with multiple steps?